### DISCUSSION TITLE
Unable to type when remoting into Mac from Windows

### DISCUSSION THREAD

**Discussion by SnippyRiffles** [rustdesk/rustdesk#9042] [General]
### Bug Description

When connecting to a Mac Client from a Remote Windows computer I'm unable to type. I can use the mouse but not the keyboard. Funny enough, when I remote into the Windows computer from the Mac, I don't have the same issue. I've tried both the legacy and map mode on the windows computer, but it makes no difference.

### How to Reproduce

Remote into Mac client from Windows machine and try typing.

### Expected Behavior

I should be able to type when connected remotely.

### Operating system(s) on local side and remote side

Windows 11 > Ventura 13.4.1

### RustDesk Version(s) on local side and remote side

1.2.7 on both

### Screenshots

N/A

**Comment by mahescho** [comment]
Like this https://github.com/rustdesk/rustdesk/discussions/8791 or directly after connect?
